sky/cli.py

```diff
@@ -4921,6 +4931,43 @@ def local_down():
             f'{colorama.Fore.GREEN}Local cluster removed.{style.RESET_ALL}')
 
 
+def change_config_port():
+    """Set the `kubernetes:ports` to be `ingress` and backup the original value."""
+    
+    if not skypilot_config.loaded_config_path():
+        logger.debug(f'Making base {CONFIG_PATH} file as it did not previously exist.')
+        default_config = {'kubernetes': {'ports': 'loadbalancer'}}
+        config_path = os.path.expanduser(CONFIG_PATH)
+        common_utils.dump_yaml(config_path, default_config)
+        skypilot_config._try_load_config()
+    
+    logger.debug(f'Creating backup at {BACKUP_PATH} of current kubernetes port forwarding option.')
+    backup_path = os.path.expanduser(BACKUP_PATH)
+    backup_port_config = skypilot_config.to_dict()['kubernetes']['ports']
+    common_utils.dump_yaml(backup_path, {'kubernetes': {'ports': backup_port_config}})
+    
+    current_path = skypilot_config.loaded_config_path()
+    logger.debug(f'Set current port forwarding option to ingress in {current_path}.')
+    updated_config = skypilot_config.set_nested(('kubernetes', 'ports'), 'ingress')
+    common_utils.dump_yaml(current_path, updated_config)
+    skypilot_config._try_load_config()
```

Comment:
Is this last `skypilot_config._try_load_config()` necessary?

Reply:
It updates the core `_dict` struct in the `skypilot_config` file, and I think it's necessary to call this function after the config file is changed to maintain consistency/coherence.

Reply:
...[continued] especially in case in the future, code is added that uses the `skypilot_config` API, to change values in the config file, and obviously in this case we want to reflect the change in the port forwarding mode we made.

Reply:
Actually, we're going to need to call this `_try_load_config` function even for immediate functionality to reflect the new port mode in `get_port_mode` in `network_utils`.

Reply:
it is necessary, confirmed :-)